app/src/androidTest/java/com/example/mindyourpet/ReminderDatabaseTest.kt

```diff
@@ -0,0 +1,83 @@
+package com.example.mindyourpet
+//./gradlew assembleDebug --stacktrace
+import androidx.room.Room
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import com.example.mindyourpet.database.PetDatabase
+import com.example.mindyourpet.database.Reminder
+import com.example.mindyourpet.database.ReminderDatabaseDao
+import kotlinx.coroutines.runBlocking
+import org.junit.After
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.io.IOException
+import kotlin.jvm.Throws
+
+@RunWith(AndroidJUnit4::class)
+class ReminderDatabaseTest {
```

Comment:
The names `PetDatabaseTest` and `RemainderDatabaseTest` are misleading. There is only a single database (with multiple tables).

Reply:
Changed to PetTableTest and ReminderTableTest.